intg/src/main/java/org/apache/atlas/model/typedef/AtlasStructDef.java

```diff
@@ -280,6 +280,7 @@ public static class AtlasAttributeDef implements Serializable {
         public static final String    ATTRDEF_OPTION_APPEND_ON_PARTIAL_UPDATE = "isAppendOnPartialUpdate";
         private final String          STRING_TRUE                             = "true";
         private final String          MULTIFIELDS                             = "multifields";
+        private boolean               isIndexPresent;
```

Comment:
do we need this flag too ? I feel if index is created , it means index is present. Cant we use same flag?